src/styleguide/components/s-theme-selector.vue

```diff
@@ -115,13 +115,9 @@
 
         this.$store.commit('session/setTheme', themeId);
 
-        if (urlParam.has('theme')) {
-          urlParam.delete('theme');
-        }
-
-        urlParam.append('theme', themeId);
+        urlParam.set('theme', themeId);
 
-        window.history.pushState({}, '', `${window.location.pathname}?${urlParam}`);
+        window.history.pushState({}, '', `${urlParam}`);
```

Comment:
todo: as mentioned in my previous comment, the `?` at the start or the string is mandatory. Else, the string is used in combination with the domain. You change leads to an url like `http://localhost:9090/theme=01`.